tests/importance_tests/test_fanova.py

```diff
@@ -71,3 +75,40 @@ def test_fanova_importance_evaluator_with_target() -> None:
     )
 
     assert param_importance != param_importance_with_target
+
+
+@pytest.mark.parametrize("inf_value", [float("inf"), -float("inf")])
+def test_fanova_importance_evaluator_with_infinite(inf_value: float) -> None:
+    # The test ensures that trials with infinite values are ignored to calculate importance scores.
+    n_trial = 10
+    trial_sampling_seed = 13
+    evaluator_seed = 67
+
+    # Calculate importance scores without inf
```

Comment:
https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#block-and-inline-comments-are-supposed-to-be-sentences

```suggestion
    # Calculate importance scores without inf.
```

Reply:
Thank you. I will update comments following the instruction.

Reply:
Brilliant. Thanks!